blt/blt.yml

```diff
@@ -17,13 +17,13 @@ drush:
   default_alias: '${drush.aliases.local}'
 modules:
   local:
-    enable: [dblog, devel, devel_debug_log, devel_generate, seckit, views_ui]
+    enable: [dblog, devel, devel_debug_log, devel_generate, seckit, views_ui, config_import_de]
     uninstall: [acquia_connector, simplesamlphp_auth, shield, config_readonly, config_ignore]
   ci:
     enable: {  }
     uninstall: [acquia_connector, shield]
   dev:
-    enable: [acquia_connector, shield]
+    enable: [acquia_connector, shield, config_import_de]
```

Comment:
Dev should be more stable than this. Do you really need it here?

Reply:
So this has kinda been what i've been thinking:
dev is 1-1 match to code for all sites - this allows me to test ALL config files
stage is direct clone of production with all config from code up to date - allows me to test issues with next production release and using import partial commands.

Thoughts on that? i'm not totally sure about it still.